### PR TITLE
Adding text to README

### PR DESCRIPTION
**Pull request by bjohnsto**

### CODE REVIEW

**Comment by bjohnsto** [comment]
This is Bruce's comment

**Comment by bjohnsto** [comment]
I saw your reply

**Comment by jshimkus-rh** [comment]
This is a PR-level comment.

File: src/tools/README.txt
Comment (jshimkus-rh) [inline src/tools/README.txt:1]: This is file-level.

File: src/tools/README.txt [hunk 1]
```diff
@@ -40,4 +40,4 @@ upgrader:	Automated upgrade mechanism.
 
 virt:		Source to our "permabit-virt" Debian package: for 
 		manipulating a virtual network interface, used in
-		some automated tests.
+		some automated tests. This is my change.
```
Comment (jshimkus-rh) [inline src/tools/README.txt:43]: old

Comment (jshimkus-rh) [inline src/tools/README.txt:43]: new

**Review by jshimkus-rh — COMMENTED** [review]
This is a review,

**Review by rhawalsh — APPROVED** [review]
This is an approval, but the merge should NOT be allowed to go in yet, because there is still a pending review that was not approved yet.

**Pull request closed without merging**